docs(menu-toggle): Updates split toggle example names. (#10930)
patternfly/patternfly-react: packages/react-core/src/components/MenuToggle/examples/MenuToggle.md

### Split toggle with labeled checkbox

You can allow users to select a toggle checkbox by clicking either the checkbox or the text label.

> To do so, pass `children` to the `<MenuToggleCheckbox>` component. When the menu toggle text is clicked, the checkbox's `onChange` callback will be triggered.

Author: ```suggestion
To link a split toggle label to the toggle's checkbox, pass both the label and the `<MenuToggleCheckbox>` component to `splitButtonOptions`. 
```

```ts file='MenuToggleSplitButtonCheckboxWithText.tsx'

```

### Split toggle with checkbox and toggle text 

> To add a text label to a split button toggle, pass `children` to the `<MenuToggle>` component.

Author: ```suggestion
To link a split toggle label to the toggle button itself, pass the label to the `<MenuToggle>` component, instead of `splitButtonOptions`.
```
